utils/tests_fetcher.py

```diff
@@ -949,18 +1065,16 @@ def infer_tests_to_run(
     if any(x in modified_files for x in ["setup.py", ".circleci/create_circleci_config.py"]):
         test_files_to_run = ["tests", "examples"]
         repo_utils_launch = True
-    # in order to trigger pipeline tests even if no code change at all
-    elif "tests/utils/tiny_model_summary.json" in modified_files:
-        test_files_to_run = ["tests"]
-        repo_utils_launch = any(f.split(os.path.sep)[0] == "utils" for f in modified_files)
```

Comment:
this is not working as `modified_files` is computed by `get_modified_python_files(diff_with_last_commit=diff_with_last_commit)`, so `tiny_model_summary` is never in that list.

My bad, sorry